pgjdbc/src/test/java/org/postgresql/test/jdbc42/SetObject310Test.java

```diff
@@ -314,18 +314,18 @@ private List<String> getZoneIdsToTest() {
   }
 
   private void instantTimestamps(Instant instant) throws SQLException {
-    String sql = "select ?::timestamp with time zone, ?::timestamp with time zone";
+    String sql = "select ?, ?";
     try (PreparedStatement ps = con.prepareStatement(sql)) {
       ps.setObject(1, instant);
       ps.setObject(2, instant, Types.TIMESTAMP_WITH_TIMEZONE);
       try (ResultSet rs = ps.executeQuery()) {
         rs.next();
-        String noType = rs.getString(1);
-        OffsetDateTime noTypeRes = parseBackendTimestamp(noType);
-        assertEquals(instant, noTypeRes.toInstant());
-        String withType = rs.getString(2);
-        OffsetDateTime withTypeRes = parseBackendTimestamp(withType);
-        assertEquals(instant, withTypeRes.toInstant());
+        String untypedOdtString = rs.getString(1);
```

Comment:
thanks, curious what does typed or untyped mean in this context ?

Reply:
@davecramer This test case which I wrote is reference the old one for `OffsetDateTime` and `LocalDateTime` . I guess it is designed to test the `setObject` method with or without SQL type specified. So I name it typed(with type) and untyped(without type).